app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,75 @@
+$flickr-pink: #ff2e92;
+$flickr-blue: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  background: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.